docs/zkapps/front-end-integration-guides/angular.mdx

```diff
@@ -0,0 +1,419 @@
+    - Files under public are served with the app, so the file itself will be available at `http://localhost:4200/lib/o1js/index.js`.
+
+```json
+"copy-libs": "mkdir -p public/lib/o1js && cp node_modules/o1js/dist/web/index.js public/lib/o1js/index.js"
+```
+
+- Add the `copy-o1js-lib` task to the build script and the start script.
+    - `"build": "npm run copy-libs && ng build"`
+    - `"start": "npm run copy-libs && ng serve"`
+- Add `public/lib` to `.gitignore`.
+
+### Load o1js with an `importmap`
+
+- Above the closing `</head>` tag in `src/index.html` add these scripts to import o1js from `public/lib/o1js`:
+
+```html
+...
+  <script type="importmap">
+	{ "imports": { "o1js": "./lib/o1js/index.js" } }
+  </script>
+  <script type="module">
+    import * as o1js from "o1js";
+    window.o1js = o1js;
+    console.log("Set the global o1js instance: ", window.o1js);
+  </script>
+</head>
+```
+
+- Now instead of importing o1js as a regular npm dependency, we declare it as a top level variable in app component knowing that it will exist in the global context of the browser at runtime. Add the following to the top of `src/app.component.ts`:
+
+```tsx
+// at the top of the file:
+import type * as o1jsTypes from 'o1js';
+declare var o1js: typeof o1jsTypes;
+```
+
+- Remove the import of o1js inside of `afterNextRender` and replace it with this:
+
+```tsx
+const {Mina, PublicKey, fetchAccount} = o1js;
+```
+
+## Running the App Locally
+
+- Congratulations! The app should work as expected when served with `npm run start` (`ng serve`).
+- Restart the application with `npm run start`.
+- Verify that `Set the global o1js instance: Module {…}` was logged in the console, indicating that o1js was successfully loaded from our public assets.
+- Verify that the current `num` on the `Add` zkapp is logged, meaning that we're successfully reading state from the contract at `B62qnTDEeYtBHBePA4yhCt4TCgDtA4L2CGvK7PirbJyX4pKH8bmtWe5` on Devnet.
+- Verify that "Compiled Add" is logged, meaning that the SDK has successfully generated a proving key for the `Add` zkapp.
+- If you're connected to Devnet and your account is funded with Devnet tokens, you should be be able to broadcast a successful transaction calling `update` on the zkapp. After a few minutes, the state change associated with the transaction will take effect on chain and you'll see `num` increase when you reload the page!
+
+## Deploying to GitHub Pages
+
+- Now we'll set the app up for deployment to GitHub pages.
+- Publish your project to a GitHub repository.
+- Run `ng deploy` and select GitHub Pages.
+- Add `baseHref` to `options` under `build` in angular.json with the name of your GitHub repository.
+    - **Do not remove the slashes!**
+
+```json
+"baseHref": "/<your-github-repo-name>/"
+```
+
+- Create a deploy script in package.json which copies the required libraries
+
+```json
+"deploy": "npm run copy-libs && ng deploy --dir=dist/<project-name>"
```

Comment:
This gave me `Error: Unknown argument: dir` at first.  I had to execute `ng deploy` with no arguments first, then select github pages from some menu, then it understood the `--dir` argument.